proxy/pkg/zdmproxy/controlconn.go

```diff
@@ -54,11 +54,11 @@ type ControlConn struct {
 	protocolEventSubscribers map[ProtocolEventObserver]interface{}
 	authEnabled              *atomic.Value
 	metricsHandler           *metrics.MetricHandler
+	protocolVersion          primitive.ProtocolVersion
```

Comment:
Do we need this? Can't we just grab the protocol version from the underlying `cqlConn`?